modules/gapi/src/backends/fluid/gfluidbackend.cpp

```diff
@@ -178,6 +179,13 @@ struct FluidFilterAgent : public FluidAgent
     virtual void setRatio(double) override { /* nothing */ }
 public:
     using FluidAgent::FluidAgent;
+    int m_window;
+
+    FluidFilterAgent(const ade::Graph &g, ade::NodeHandle nh):FluidAgent(g, nh)
+    {
+        m_window = GConstFluidModel(g).metadata(nh).get<FluidUnit>().window;
+    }
+    virtual ~FluidFilterAgent() = default;
```

Comment:
FluidAgent has a virtual destructor, this one should be removed

Reply:
Applied.